### ISSUE TITLE
docker-gen.sh needs to be fixed

### ISSUE THREAD

**Issue by svetasmirnova** [percona/pxc-docker#1]
See comments to https://www.percona.com/blog/2015/06/30/playing-percona-xtradb-cluster-docker/

Suggested fix:

diff --git a/docker-bld/docker-gen.sh b/docker-bld/docker-gen.sh
index 71432ef..b29bb9e 100755
--- a/docker-bld/docker-gen.sh
+++ b/docker-bld/docker-gen.sh
@@ -25,8 +25,8 @@ numcp=$(( numcp-1 ))
 echo "
 FROM centos:centos7
 MAINTAINER Raghavendra Prabhu raghavendra.prabhu@percona.com
-RUN curl -s http://jenkins.percona.com/dev-repo/percona-dev.repo > /etc/yum.repos.d/percona-dev.repo
-RUN yum install -y http://epel.check-update.co.uk/7/x86_64/e/epel-release-7-5.noarch.rpm
+RUN curl -s http://jenkins.percona.com/yum-repo/percona-dev.repo > /etc/yum.repos.d/percona-dev.repo
+RUN yum install -y http://epel.check-update.co.uk/7/x86_64/e/epel-release-7-6.noarch.rpm
 RUN yum install -y http://www.percona.com/downloads/percona-release/redhat/0.1-3/percona-release-0.1-3.noarch.rpm
 RUN yum install -y which lsof libaio compat-readline5 socat percona-xtrabackup perl-DBD-MySQL perl-DBI rsync openssl098e eatmydata pv qpress gzip openssl
 RUN yum install -y bzr automake gcc  make  libtool autoconf pkgconfig gettext git scons    boost_req boost-devel libaio openssl-devel  check-devel gdb perf
@@ -35,7 +35,7 @@ RUN yum install -y coreutils grep procps
 RUN git clone --depth=1 -b $branch http://github.com/percona/percona-xtradb-cluster
 WORKDIR /percona-xtradb-cluster
 RUN cmake -DBUILD_CONFIG=mysql_release -DDEBUG_EXTNAME=OFF -DWITH_ZLIB=system  -DWITH_SSL=system -DCMAKE_INSTALL_PREFIX="/usr"   .
-RUN make -j$numcp
+RUN make -j 1
 RUN make install
 WORKDIR /
 RUN git clone --depth=1 https://github.com/percona/galera

**Comment by cyb3ralbert** [comment]
yes. thank you. method was understood. some files is just 404. by the way,
nowadays epel-release-7-8.noarch.rpm

**Comment by hrvojem** [comment]
For EPEL using:
https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
and
https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
instead of fixed version would probably be better.
